docs/wsrep-system-index.md

```diff
@@ -666,20 +666,21 @@ when the node tries to join again but the change will be undone.
 | Dynamic:       | Yes                 |
 | Default Value: | ``OFF`` |
 
-This variable has been announced as deprecated in `5.7.24-31.33`. Defines whether the node should use transparent handling
-of preordered replication events (like replication from traditional source).
-By default, this is disabled.
+This variable has been announced as deprecated in `5.7.24-31.33`.
+
+Defines whether the node should use transparent handling of preordered replication events (like replication from traditional source). By default, this variable is disabled.
 
-If you enable this variable, such events will be applied locally first
-before being replicated to other nodes in the cluster.
-This could increase the rate at which they can be processed,
-which would be otherwise limited by the latency
-between the nodes in the cluster.
+If you enable this variable, preordered replication events are first applied locally before the events are replicated to other nodes. This behavior could increase the rate at which they can be processed, which would be otherwise limited by the latency between the nodes in the cluster.
 
-Preordered events should not interfere with events that originate on the local
-node. Therefore, you should not run local update queries on a table that is
+Preordered events should not interfere with events that originate on the local node. Do not run local update queries on a table that is
 also being updated through asynchronous replication.
 
+#### Limitations
```

Comment:
@kamil-holubicki Should we also mention that this feature must to be used only for quick loading of data and must be disabled after loading the data?

Reply:
Discussed with Kamil and added.